app/assets/stylesheets/practices.scss

```diff
@@ -0,0 +1,57 @@
+.profile {
+  margin: none;
+  border-bottom: 1px solid #ddd;
+  padding: 10px;
+  background-color: #fff;
+  .name {
+    h3 {
+      margin: 10px;
+    }
+  }
+}
+
+.practices-list {
+  margin: 20px 5px;
+  list-style-type: none;
+  .practice {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    border-radius: 3px;
+  }
+}
+
+.practice {
+  .title {
+    margin: none;
+    border-bottom: 1px solid #ddd;
+    .fa.fa-check-square-o {
+      text-align: center;
+      float: left;
+      margin: 10px
+    }
+    h3 {
+      margin: 10px;
+      font-weight: 900;
+    }
+  }
+  .daily-progress {
+    padding: 10px;
+    border-bottom: 1px solid #ddd;
+  }
+  .total, .longest-streak {
+    border-bottom: 1px solid #ddd;
+  }
+  .total, .longest-streak, .current-streak {
+    padding: 10px;
+    .text-muted {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3